.github/workflows/on_pull-request_closed.yaml

```diff
@@ -12,6 +12,7 @@ env:
 jobs:
   remove-temp-image:
     runs-on: ubuntu-22.04
+    if: github.repository == 'zapier/kubechecks'
```

Comment:
I think we just need to add a `continue-on-error: true` here, and the next place